CondTools/CTPPS/plugins/PPSTimingCalibrationLUTAnalyzer.cc

```diff
@@ -0,0 +1,50 @@
+/****************************************************************************
+ *
+ * This is a part of TOTEM/PPS offline software.
+ * Author:
+ *   Laurent Forthomme (laurent.forthomme@cern.ch)
+ *   Christopher Misan
+ *
+ ****************************************************************************/
+
+#include "FWCore/Framework/interface/Frameworkfwd.h"
+#include "FWCore/Framework/interface/one/EDAnalyzer.h"
+#include "FWCore/Framework/interface/Event.h"
+#include "FWCore/Framework/interface/EventSetup.h"
+#include "FWCore/Framework/interface/ESHandle.h"
```

Comment:
I think this is not needed, you get the data with `getData()` so the `ESHandle` is not used

Reply:
agreed